baseplate/context/redis.py

```diff
@@ -72,8 +73,8 @@ class RedisContextFactory(ContextFactory):
     def __init__(self, connection_pool):
         self.connection_pool = connection_pool
 
-    def make_object_for_context(self, name, server_span):
-        return MonitoredRedisConnection(name, server_span, self.connection_pool)
+    def make_object_for_context(self, name, span):
```

Comment:
I like this but it _is_ a breaking change if anyone called this with kwargs so we'll have to watch out for that.

Reply:
gp. I've done a quick pass on code search in our org and don't see any uses with kwargs, but will note this as a breaking change in the changelog.